de.fu_berlin.inf.dpp.intellij/src/de/fu_berlin/inf/dpp/intellij/editor/EditorManager.java

```diff
@@ -490,10 +489,8 @@ public EditorManager(ISarosSessionManager sessionManager,
         this.annotationManager = annotationManager;
         this.fileReplacementInProgressObservable = fileReplacementInProgressObservable;
 
-        documentListener = new StoppableDocumentListener(this,
-                virtualFileConverter);
-        fileListener = new StoppableEditorFileListener(this,
-                virtualFileConverter, annotationManager);
+        documentListener = new StoppableDocumentListener(this);
```

Comment:
I do not get this, I see this frequently currently that lines are affected by the formatter that should not be affected.

Reply:
I am not sure I know what you mean. Could you elaborate?

If you are suggesting that the code formatter is behaving inconsistently, you are right. The profiles set up for IntelliJ and Eclipse don't match (lead to a different formatting). We had a discussion on how to proceed, but the consensus for now was to wait until we switch to a newer Eclipse version as (apparently) the code formatting handling changed in Eclipse at some point. After that, there are a couple option how to proceed. For example, we could use a plugin for IntelliJ that allows to use an Eclipse code formatting setting. (This went hand in hand with the discussion of whether we should raise the allowed line length to 120 characters.)

But, if this is what you mean, I still don't understand what it has to do with the quoted lines. Or do you mean the lines before the changes?